hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/MultiObjectDeleteException.java

```diff
@@ -41,7 +43,7 @@ public class MultiObjectDeleteException extends S3Exception {
   private final List<S3Error> errors;
 
   public MultiObjectDeleteException(List<S3Error> errors) {
-    super(builder().message(errors.toString()));
```

Comment:
why do we need to do this?

Reply:
S3AUtils.translateException checks the status code before trying to translate this exception